burn-train/src/metric/dashboard/base.rs

```diff
@@ -179,12 +193,12 @@ where
 }
 
 trait DashboardNumericMetric<T>: Send + Sync {
-    fn update(&mut self, item: &LearnerItem<T>) -> (MetricEntry, f64);
+    fn update(&mut self, item: &LearnerItem<T>, metadata: &MetricMetadata) -> (MetricEntry, f64);
```

Comment:
for consistency, should use `LearningRate` instead of 64.

Reply:
This is not the LearningRate, this is any numeric value that can be plotted on the terminal.